docs/gke/configs/cluster-kubeflow.yaml

```diff
@@ -21,33 +21,54 @@ resources:
   # 1. Use a single template (.jinja file for all resources) or
   # 2. Create two separate deployments and launch the boot strapper
   # after the cluster is created.
-  # 
-  # Two separate deployments doesn't make much sense; we could just use 
+  #
+  # Two separate deployments doesn't make much sense; we could just use
   # kubectl at that point. So we put all resources in a single deployment.
 - name: kubeflow
   type: cluster.jinja
   properties:
     zone: us-east1-d
+    # Set this to v1beta1 to use beta features such as private clusters,
+    apiVersion: v1
```

Comment:
v1beta1 or v1?

Reply:
I want to set a default value of `v1`. If the end-user wants to leverage beta features, they can set this to v1beta1

Reply:
Is this the GKE API version? Should we call it gkeApiVersion?

Reply:
Yes. Updated